packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/index.md

```diff
@@ -113,66 +114,119 @@ For password-only authentication, you need to update the password authenticator
+2. [Add a new Global session policy for the group](#add-a-new-global-session-policy-for-password-optional-users).
+3. [Add a new authentication policy for the group](#add-a-new-authentication-policy-for-password-optional-users).
+4. Ensure that password-optional users will never fall through to the default policy. Default policy should always have a password as a required authenticator.
+5. Explicitly exclude your main admin account from any further password-optional policies you create.
 
-#### Create a password-optional authenticator enrollment policy
+> **Note**: See also [Set up password-optional sign-in experience](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/password-optional/password-optional-disabled.htm)
+
+#### Common password-optional setup tasks
+
+##### Set up the email authenticator
 
 1. Open the **Admin Console** for your org.
+2. Choose **Security** > **Authenticators** to show the available authenticators.
+3. On the **Setup** tab, locate the **Email** authenticator, and then select **Actions** > **Edit**.
+4. Set **This authenticator can be used for** to **Authentication and recovery**.
+5. Click **Save**.
+
+##### Create a group for password-optional users
+
+1. Choose **Directory** > **Groups**.
+2. Click **Add Group**.
+3. Give the group a name. For example, "Password-optional Users".
+4. Click **Save**.
+
+#### Enable password-optional user sign-up
+
+To ensure that only specific app integrations can let users sign up without a password, do the following:
+
+##### Create a profile enrollment policy for password-optional users
+
+A profile enrollment policy determines the minimum information required from a user to create an account, and how the user should verify their identity before creating their account.
+
+1. Choose **Security** > **Profile Enrollment** to show the existing enrollment policies.
+2. Click **Add Profile Enrollment Policy**.
+3. Locate the **Profile Enrollment** section of the policy and click **Edit**.
+4. Set **Self-service registration** to **Allowed**.
+5. Verify that **Required before access is granted** is checked for **Email Verification**.
+6. Set **Add the user to group** to the group you just made for password-optional users.
+7. Click **Save**.
+8. Click **Manage apps**.
+9. Click **Add an App to This Policy**.
+10. Locate your app integration in the list and click **Apply** next to it.
+11. Verify that the app is now in the list of Apps using the new Profile Enrollment Policy.
+
+#### Enable password-optional user sign-in
+
+To ensure that only password-optional users can sign in without a password and everybody else is appropriately prompted for it, do the following:
+
+##### Create a password-optional authenticator enrollment policy
+
+An authenticator enrollment policy determines which authenticators must challenge a user before they are successfully logged in. In this case, email is set to Required, while all the other authenticators are set to Optional.
```

Comment:
```suggestion
An authenticator enrollment policy determines which authenticators must challenge a user before they are successfully signed in. In this case, email is set to **Required**, while all the other authenticators are set to **Optional**.
```

Reply:
Grr. There's always one log \ sign miss. Thank you!